tests/Avalonia.Benchmarks/Themes/ThemeBenchmark.cs

```diff
@@ -41,7 +41,7 @@ public bool InitFluentTheme(FluentThemeMode mode)
         [Benchmark]
         [Arguments(SimpleThemeMode.Dark)]
         [Arguments(SimpleThemeMode.Light)]
-        public bool InitDefaultTheme(SimpleThemeMode mode)
+        public bool InitSimpleTheme(SimpleThemeMode mode)
```

Comment:
This may have intentionally kept the old `InitDefaultTheme` name. If so, I can revert it (or one of the maintainers if they get to it before me).

Reply:
No, it was missed too.

Reply:
Ok, thanks for confirming